tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj

```diff
@@ -14,6 +14,13 @@
     <IntermediateOutputPath>$(ArtifactsDir)obj/$(MSBuildProjectName)/$(Configuration)/</IntermediateOutputPath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="Xunit.TestFramework">
```

Comment:
For my understanding, why is this needed and how will this work?

Reply:
It installs the customized xUnit TestFramework.
The only customization there is that it installs the multithreaded console capture.
It's a way to guarantee the console is captured before any code in the assembly runs:
https://github.com/dotnet/fsharp/blob/5e7e1e8eec0942853c427dd1dd2c8503fbe7e1fb/tests/FSharp.Test.Utilities/Utilities.fs#L100-L102

Reply:
Oh I see. Alright, if we decide to stay with this, it's worth documenting somewhere :)